Dockerfile

```diff
@@ -1,38 +1,22 @@
-FROM debian:buster-slim
-
-# Debian Base to use
-ENV DEBIAN_VERSION buster
-
-# initial install of av daemon
-RUN echo "deb http://http.debian.net/debian/ $DEBIAN_VERSION main contrib non-free" > /etc/apt/sources.list && \
-    echo "deb http://http.debian.net/debian/ $DEBIAN_VERSION-updates main contrib non-free" >> /etc/apt/sources.list && \
-    echo "deb http://security.debian.org/ $DEBIAN_VERSION/updates main contrib non-free" >> /etc/apt/sources.list && \
-    apt-get update && \
-    apt-get upgrade -y && \
-    apt-get install ruby-full -y && \
+FROM clamav/clamav:stable
+
+USER root
+
+# initial install of packages
+RUN apk update && \
+    apk add ruby && \
+    apk add ca-certificates && \
     gem install faraday -v >= 1.9.3 && \
     gem install sentry-raven && \
-    gem install rufus-scheduler && \
-    apt-get install -y ca-certificates && \
-    DEBIAN_FRONTEND=noninteractive apt-get install --no-install-recommends -y -qq \
-        clamav \
-        clamav-daemon \
-        clamav-freshclam \
-        wget && \
-    apt-get clean && \
-    rm -rf /var/lib/apt/lists/*
-
-COPY --from=ghcr.io/ministryofjustice/hmpps-clamav-freshclammed:latest /var/lib/clamav/main.cvd /var/lib/clamav/main.cvd
-COPY --from=ghcr.io/ministryofjustice/hmpps-clamav-freshclammed:latest /var/lib/clamav/daily.cld /var/lib/clamav/daily.cld
-COPY --from=ghcr.io/ministryofjustice/hmpps-clamav-freshclammed:latest /var/lib/clamav/bytecode.cvd /var/lib/clamav/bytecode.cvd
+    gem install rufus-scheduler
 
-RUN chown clamav:clamav /var/lib/clamav/*.cvd
 # permission juggling
+RUN chown clamav:clamav /var/lib/clamav/*.cvd
 RUN mkdir /var/run/clamav && \
     chown clamav:clamav /var/run/clamav && \
     chmod 750 /var/run/clamav
 
-# av configuration update
+    # av configuration update
```

Comment:
Probably of no consequence but if I was being really picky I'd indent the comment :D